app/models/step_by_step_page.rb

```diff
@@ -78,7 +78,10 @@ def mark_as_scheduled
   end
 
   def mark_as_unscheduled
-    update_attribute(:status, "draft")
+    update_attributes(:status => "draft",
```

Comment:
Nitpick: I'm surprised the linter didn't pick up on this, but symbols can used here instead of hash rockets.

`update_attributes` is an alias for `update` which is what's used in other places in this file, so perhaps this should be updated to match.